edx/frontend-app-admin-portal
# ENT-3694: Batch subscriptions UI

Multiple Subscription Admin Experience
-
### Ticket
[ENT-3694](https://openedx.atlassian.net/browse/ENT-3694)

### Testing Instructions

I suggest configuring test data through LMS and license-manager Django admin as follows:
- Enterprise Customer A
    - 1 SubscriptionPlan that has not had any licenses assigned or activated
- Enterprise Customer B
    - 2 SubscriptionPlans

Use Customer A to test the redirect from the list view to the detail view when there is only one available subscription plan. While on the detail view you will be able to test that the zero-state functionality is still in tact since none of the licenses for Customer A's subscription plan have been assigned or activated. 

Use Customer B to test the list view which displays a card for each subscription plan available to the given customer.

Significant refactoring was done to the Subscriptions Management Page to make this possible. Poking around to ensure all other functionality for the detail view is still in tact is highly encouraged. I also invite you to critique the new directory structure within the subscriptions folder.

Note: A bug exists in which the overview (and sometimes detail) context doesn't actually refresh after a successful license assignment or revocation. See [comment](https://github.com/edx/frontend-app-admin-portal/pull/390#issuecomment-745460170) below for details. My recommendation is to address that bug in a separate PR and avoid any further delays.
